resources/views/checkout/partials/address-cards-popup.blade.php

```diff
@@ -1,7 +1,7 @@
 <input type="checkbox" id="popup" class="peer hidden"/>
 <div class="fixed inset-0 opacity-0 transition z-50 flex justify-center items-center pointer-events-none peer-checked:opacity-100 peer-checked:pointer-events-auto">
     <x-rapidez-ct::sections class="relative z-10">
-        <x-rapidez-ct::card.inactive>
+        <x-rapidez-ct::card.inactive class="max-h-dvh overflow-y-auto scrollbar-hide relative xl:max-h-[calc(100dvh-44px)]">
```

Comment:
I think you can just use max-h-full, maybe need to give it to his parent as well since the most upper parent is a fixed inset-0 element

Reply:
Check 👍